### DISCUSSION TITLE
HOST in .env file

### DISCUSSION THREAD

**Discussion by pierreberchtold** [flyntwp/flynt#487] [Ideas]
Would it make sense to be able to define the host in the .env file like in flynt 1.4?

It would mainly require a change in `vite.config.js` on line 10:
`const env = loadEnv(mode, process.cwd(), '')
const envHost = env.HOST || host`

And line 32 `{ dest, host }` should be replaced with `{ dest, host: envHost }`

@timohubois what are your thoughts on that? Is there a reason what this has been removed from version 1.4?

**Comment by timohubois** [comment]
@pierreberchtold thank you very much for your idea! It seems that you are talking about the current Flynt next branch, correct?

Unfortunately at the moment I don't understand exactly what you mean or what exactly you want to achieve or optimise where and how?

Would be really happy to get more details and where exactly you see what kind of benefit?

**Reply by pierreberchtold** [reply]
Hey @timohubois! yes, I was talking about the next branch.
I wanted to move the local domain to a `.env' file instead of hardcoding it. That way you can have different local domains for different developers. 
But I now realise that this was also hardcoded in flynt version 1. So maybe I am the only one who feels like this belongs in the environment settings =)

**Reply by steffenbew** [reply]
@pierreberchtold Thanks for the suggestion! 

We implemented an option to provide the host in an `.env` file with Flynt v2.0:
`const host = env.VITE_DEV_SERVER_HOST || wordpressHost`

see https://github.com/flyntwp/flynt/blob/master/vite.config.js#L29
